new/MDDL002.xml

```diff
@@ -0,0 +1,445 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MDDL002" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Tǝrgʷāme malʾǝkt za-bǝṣuʿ wa-kǝbur Ṗāwlos za-ṣaḥafa la-ʿƎbrāwǝyān, Qerǝllos</title>
+                <editor role="generalEditor" key="AB"/>
+                  <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt><p>Digitized and catalogued within the framework of the project Documenting an 
```

Comment:
this probably has to be changed, as the MS was digitized much earlier, within the framework of EAP336 project - but it was not submitted among the EAP mss, so I am not sure; possibly we should simply write that it was digitized by @Mersha-Alehegne @MershaMengistie in 2010-2011 and catalogued within the framework of the ʾAbǝnnat Tǝmhǝrt project